cmd/mattermost/commands/team.go

```diff
@@ -339,3 +353,43 @@ func archiveTeamCmdF(command *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func modifyTeamCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Shutdown()
+	username, err := command.Flags().GetString("username")
+	if username == "" || err != nil {
+		return errors.New("Username is required.")
+	}
+
+	public, _ := command.Flags().GetBool("public")
+	private, _ := command.Flags().GetBool("private")
+
+	if public == private {
+		return errors.New("You must specify only one of --public or --private")
+	}
+
+	team := getTeamFromTeamArg(a, args[0])
+	if team == nil {
+		return errors.New("Unable to find team '" + args[0] + "'")
+	}
+
+	if !(team.Type == model.TEAM_OPEN || team.Type == model.TEAM_PRIVATE) {
```

Comment:
@grundleborg Just like to know about the "I" (or TEAM_INVITE) type. Will it remain as is and will not be possible to modify after team creation?